vars/node811BuilderContainerTemplate.groovy

```diff
@@ -2,7 +2,7 @@ def call() {
   return [
     containerTemplate(
       name: 'node811-builder',
-      image: 'cypress/base',
+      image: 'cypress/base:12.14',
```

Comment:
Use this https://github.com/agiledigital/node12-cypress-jenkins-library instead of upping the version.